pymcp/runtime/handlers/listing.py

```
@@ -0,0 +1,60 @@
+    result_key: str,
+    filter_method: str | None = None,
+) -> DispatchResult:
+    authorizer = getattr(getattr(ctx.app, "state", None), "authorizer", None)
+    principal = getattr(ctx.session, "principal", None)
+    if authorizer and filter_method:
+        filter_fn = getattr(authorizer, filter_method, None)
+        if callable(filter_fn):
+            try:
+                items = list(filter_fn(principal, items))
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)